doc/tutorials/ostro-oic-tutorial.md

```diff
@@ -0,0 +1,1030 @@
+# Overview
+
+In this tutorial, we'll show how to develop a simple application using
+Soletta. Although simple, this application will show some important
+features of Soletta, like OIC and HTTP communication, besides GPIO
+interaction.
+Initially, we are going to run the application on desktop [1], but
+later we're going to show how to run it on devices, such as [MinnowBoard
+MAX](http://wiki.minnowboard.org/MinnowBoard_Wiki_Home).
+
+## Step 0
+
+Soletta must be installed on developer's computer. One can use a package
+provide for his or her distro, or [build Soletta](https://github.com/solettaproject/soletta/wiki/How-to-start).
+
+## Step 1
```

Comment:
for step-1 I'd start with ideation of the solution. You can describe it in text and then add a graphical representation about it, close to FBP, but no mention of FBP at that point.

Then, when you get into actual implementation you mention FBP was chosen since it is closer to that way of thinking